File: DSA/pranavdixit_590014418/Day_16_Question_2.py
def count_occurrences(arr, target):
    count = 0
    for i in arr:
        if i == target:
            count += 1
    return count

print(count_occurrences([1, 2, 3, 2, 2, 4], 2))
print(count_occurrences([5, 1, 5, 5, 3, 5, 7], 5))
print(count_occurrences([10, 20, 30, 10, 40], 10))

File: DSA/Tarakshgoyal_590011425/day16question2.cpp
#include <iostream>
using namespace std;

int countOccurrences(int arr[], int n, int target) {
    int count = 0;
    for (int i = 0; i < n; i++) {
        if (arr[i] == target)
            count++;
    }
    return count;
}

int main() {
    int arr[1000], n, target;

    cout << "Enter number of elements: ";
    cin >> n;

    cout << "Enter " << n << " integers:\n";
    for (int i = 0; i < n; i++) {
        cin >> arr[i];
    }

    cout << "Enter the target number to count: ";
    cin >> target;

    int count = countOccurrences(arr, n, target);

    cout << "The number " << target << " appears " << count << " times in the array." << endl;

    return 0;
}
